scio-core/src/main/scala/com/spotify/scio/values/PairSCollectionFunctions.scala

```diff
@@ -624,6 +626,30 @@ class PairSCollectionFunctions[K, V](val self: SCollection[(K, V)])
     new MapSideInput[K, V](o)
   }
 
+  def asSparkeySideInput: SideInput[Map[String, String]] = {
+    val f = self.groupBy(_ => ()).values
+      .map(_.map(kv => (kv._1.toString, kv._2.toString))).map { iter =>
```

Comment:
👍 although for the `.values.map` though not sure what you mean by fuse? If I do `.mapValues` I would still need to do `.values` at the end anyway to have `SCollection[String]` of the file path, so I figured it makes since to get rid of the key up front.